ActionNetworkContactImporter.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+require_once("AbstractContactImporter.php");
+
+require __DIR__ . '/vendor/autoload.php';
+
+use Ekino\HalClient\HttpClient\FileGetContentsHttpClient;
+use Ekino\HalClient\EntryPoint;
+
+use Ekino\HalClient\HttpClient\HttpClientInterface;
+use Ekino\HalClient\Resource;
+
+use GuzzleHttp\Client;
+
+//TODO: Configure my include path to allow civicrm.api.php for civicrm_api3
```

Comment:
This raises the question of how we intend to bootstrap CiviCRM.  I realize this is a toy implementation, so the answer here is likely different from the finished product.  You can use `eval(cv('php:boot', 'phpcode'));` for now, though obviously `eval()` has no place in production code.